lib/cisco_node_utils/cmd_ref/acl_v4.yaml

```diff
@@ -0,0 +1,25 @@
+# Command Reference Common ACL
```

Comment:
You can reduce the redundancy here and collapse `acl_v4.yaml` and `acl_v6.yaml` into a single file `acl.yaml` that handles both.